_gsocproposals/2018/proposal_LHCbHEPDrone.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Extension and characterisation of the LHCb data isolation drone neural networks
+layout: gsoc_proposal
+project: LHCb
+year: 2018
+organization: CERN, NWO, STFC
```

Comment:
are you sure about STFC or don't you rather mean University of Edinburgh?

Reply:
STFC provide the funding for Konstantin, and NWO provides the funding for me. However, we are LHCb authors through the University of Edinburgh and Nikhef respectively. But on the other hand, NWO and STFC are the funding sources that are acknowledged in all LHCb papers as well as our reduced author list ones. So to bottom line it the institutions we go to are the University of Edinburgh and Nikhef but the organisations that provide the funding are the ones that I listed with the keyword.

Reply:
Just to add this is now changed to Nikhef and UoE.